src/riot-rs-embassy-common/src/gpio.rs

```diff
@@ -20,9 +20,10 @@ impl From<Level> for bool {
 
 impl From<bool> for Level {
     fn from(boolean: bool) -> Self {
-        match boolean {
-            false => Level::Low,
-            true => Level::High,
+        if boolean {
```

Comment:
This turns around the result, is that intended?

Reply:
I wonder if we could encode one `From` direction with the other, so there's only one source of truth?